src/lib/pages/backoffice/Acquisition/Order/OrderDetails/OrderLiteratureRequest.js

```diff
@@ -0,0 +1,94 @@
+import { PropTypes } from 'prop-types';
+import React from 'react';
+import { withCancel } from '@api/utils';
+import { documentRequestApi } from '@api/documentRequests/documentRequest';
+import { Link } from 'react-router-dom';
+import { BackOfficeRoutes } from '@routes/urls';
+import { Message } from 'semantic-ui-react';
+import { Loader } from '@components/Loader';
+import _ from 'lodash';
```

Comment:
this way you import everything from lodash. 
please check the rest of the application code to see the correct import of the `isEmpty` function